### ISSUE TITLE
Crash in base::i18n::InitializeICU

### ISSUE THREAD

**Issue by iefremov** [brave/brave-browser#12262]
```
[ 00 ] base::i18n::InitializeICU()
[ 01 ] RtlUnwindEx
[ 02 ] content::ContentMainRunnerImpl::Initialize(content::ContentMainParams const &)
[ 03 ] base::allocator::WinHeapMalloc(unsigned __int64)
[ 04 ] malloc
[ 05 ] base::internal::LockImpl::Lock()
[ 06 ] RtlUnwindEx
[ 07 ] 0x7ffb73f6e800
[ 08 ] RtlUnwindEx
[ 09 ] mojo::core::InitializeCore()
[ 10 ] service_manager::Main(service_manager::MainParams const &)

```
https://brave.sp.backtrace.io/p/brave/debug?filters=(_deleted%3D0%2C(ver%2Cregex%2C%228%5B5%7C6%7C7%7C8%7C9%5D.*%22)%2Cptype%3Dbrowser)&fingerprint=b3787639f20b2c159b8d8e7d9986e0370f18f26868cbb6e38968537ff8e20573&debug=(b758bd,0,0)

**Comment by iefremov** [comment]
Looks like windows specific
```
[ 00 ] base::i18n::InitializeICU()
[ 01 ] RtlUnwindEx
[ 02 ] content::ContentMainRunnerImpl::Initialize(content::ContentMainParams const &)
[ 03 ] RtlUnwindEx
[ 04 ] malloc
[ 05 ] base::internal::LockImpl::Lock()
[ 06 ] 0x7ffec73ce800
[ 07 ] RtlUnwindEx
[ 08 ] mojo::core::InitializeCore()
[ 09 ] service_manager::Main(service_manager::MainParams const &)
[ 10 ] base::WideToUTF8(base::BasicStringPiece<std::__1::basic_string<wchar_t,std::__1::char_traits<wchar_t>,std::__1::allocator<wchar_t> > >)
[ 11 ] base::CommandLine::GetSwitchValueASCII(base::BasicStringPiece<std::__1::basic_string<char,std::__1::char_traits<char>,std::__1::allocator<char> > > const &)
```

https://brave.sp.backtrace.io/p/brave/debug?filters=(_deleted%3D0%2C(ver%2Cregex%2C%228%5B5%7C6%7C7%7C8%7C9%5D.*%22)%2Cptype%3Dbrowser)&fingerprint=bd4aed75b0dd7a6f5847cd5a17962200a1c4d66c26b034e3da2bfefe234d2af5&debug=(b6cf6e,0,0)

**Comment by iefremov** [comment]
https://brave.sp.backtrace.io/p/brave/explore?time=all&filters=((fingerprint%2Cregex%2C%20b3787639f20b2c159b8d8e7d9986e0370f18f26868cbb6e38968537ff8e20573%7C7a03d39b0d0f708cc0f8cdbbc9a2b7d5108eef3679c56812ec3e1bedff3f36a6%7C6227bde443c4b4fbc1af3e0cb9d6f5ca5138a8ea071b8eec1de0b25eddf0914b%7Ca3cba17e5214450e11f27bac91f5f4a745645bb59e791c9898b1d0a49353a103%7Ccec679cd5b9dd4f35b19ae6e017061f11f49ce9af59a084f825b76fe45ea1c10%7Cfc747f4dfe62e8c5092c6798b35a29ee7bce61d1bb7d8cd0c0287ea8ca656a8e%7Ce324006f90b131d2ff1235e412a297fb8b1c3a833e1b2d97665e6d8b81871346%7Cbd4aed75b0dd7a6f5847cd5a17962200a1c4d66c26b034e3da2bfefe234d2af5%7Ce463431cf738aa0e6956c781aee56239b88cd21fe7b02882c778522e1bf65f1c))&aggregations=((fingerprint%3Bfirst_seen%2Cmin)%2C(uname.sysname%2Cdistribution%2C3)%2C(callstack%2Chead))&

**Comment by rebron** [comment]
@iefremov Still an issue with latest 1.20.x?

**Comment by iefremov** [comment]
@rebron happens in 88.1.21.28

https://brave.sp.backtrace.io/p/brave/explore?filters=(ver%3D%2288.1.21.28%22%2C_deleted%3D0%2C(ver%2Cregex%2C%2288%7C89%7C90.*%22)%2Cptype%3Dbrowser)&aggregations=((ver%2Cdistribution)%2C(callstack%2Chead))&fingerprint=0000000000000000000000000000000000000000000000000000000000000449&

**Comment by iefremov** [comment]
Seems it was a consequence of https://github.com/brave/brave-browser/issues/7205, so finally closing this as the issue resolved itself :)

**Comment by iefremov** [comment]
Closed by mistake, this is a dupe of https://github.com/brave/brave-browser/issues/11055 that is not fixed yet

**Comment by kjozwiak** [comment]
> Closed by mistake, this is a dupe of #11055 that is not fixed yet

Closing the above as it's a duplicate of https://github.com/brave/brave-browser/issues/11055 as per the above. There's no need to leave this one opened as work will eventually be done via https://github.com/brave/brave-browser/issues/11055.
